dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -71,7 +71,6 @@
 
         #schooldropdown-block.itemblock{style: "display: none;"}
           = render partial: 'shared/school_info_registration', locals: {form_name: "user[school_info_attributes]", existing_school_info: nil, suppress_scrolling: true}
-          .error_in_field
```

Comment:
ah okay.  I think at one point we possibly had a single error for the entire dropdown.